client/components/Coverages/CoverageItem.tsx

```diff
@@ -120,8 +123,13 @@ export class CoverageItemComponent extends React.Component<IProps, IState> {
             userAssigned: null,
             displayContentType: '',
             coverageDateText: '',
+            addedToWorkflow: false,
```

Comment:
what about using `addedToWorkflow: coverage.workflow_status === COVERAGES.WORKFLOW_STATE.ACTIVE` so you don't need a block below?